ai_bridge/models/ai_bridge.py

```diff
@@ -0,0 +1,130 @@
+# Copyright 2025 Dixmit
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+import json
+
+from odoo import api, fields, models
+from odoo.tools.safe_eval import safe_eval
+
+
+class AiBridge(models.Model):
+
+    _name = "ai.bridge"
+    _description = "Ai Bridge Configuration"
+    _order = "sequence, id"
+
+    sequence = fields.Integer(
+        default=10,
+    )
+    name = fields.Char(required=True)
+    active = fields.Boolean(default=True)
+    description = fields.Text()
+    model_id = fields.Many2one(
+        "ir.model",
+        string="Model",
+        required=True,
+        ondelete="cascade",
+        help="The model to which this bridge is associated.",
+    )
+    model = fields.Char(related="model_id.model", string="Model Name")
+    domain = fields.Char(
+        string="Filter", compute="_compute_domain", readonly=False, store=True
+    )
+    execution_ids = fields.One2many("ai.bridge.execution", "ai_bridge_id")
+    execution_count = fields.Integer(
+        compute="_compute_execution_count",
+    )
+    url = fields.Char(
+        string="URL",
```

Comment:
Do we validate the URL?
tools.validate_url()

Reply:
No, we don't validate it. However, tools.validate_url will only add http at the begining. I am not sure if this fulfills ours needs :thinking: